lib/signs_ui_web/controllers/messages_html.ex

```diff
@@ -0,0 +1,5 @@
+defmodule SignsUiWeb.MessagesHTML do
```

Comment:
How exactly do these HTML modules work? (`MessagesHTML`, `ScuHTML`, `SingleSIgnHTML`, `UnauhorizedHTML`). I see a few of the other ones (`Layouts` and `ErrorHTML`) being explicitly used, but I couldn't find references to the other ones. 

Are they just used to generate each of the folder templates under the hood somewhere? Or maybe I missed them being referenced somewhere.